src/libs/actions/IOU.ts

```diff
@@ -1621,11 +1621,6 @@ function createSplitsAndOnyxData(
             key: `${ONYXKEYS.COLLECTION.TRANSACTION}${splitTransaction.transactionID}`,
             value: {pendingAction: null},
         },
-        {
```

Comment:
We should also add this to optimisticData as mentioned in your alternate change as a fail safe.